app/controllers/albums_controller.rb

```diff
@@ -187,7 +187,7 @@ def update
   end  
 private
   def authenticate
```

Comment:
Well, as it is now - we wanted to specify which rule was used. For example in the album controller we wanted to check if user was ```current_user.moderator?(:album)```.

If we change it to application controller with a variable for the ```:rule_name```it can work with the_role.